### DISCUSSION TITLE
CNCF TAG Runtime Discussion/Presentation

### DISCUSSION THREAD

**Discussion by rajaskakodkar** [rootless-containers/usernetes#304] [General]
Hello Usernetes maintainers,

As part of [CNCF TAG Runtime](https://github.com/cncf/tag-runtime), I am reaching out to you to present/discuss Usernetes at one of our meetings.

It would be great to learn about the "Generation 2" of Usernetes: Rootless Kubernetes, after your [last update in TAG Runtime meeting](https://docs.google.com/document/d/1k7VNetgbuDNyIs_87GLQRH2W5SLgjgOhB6pDyv89MYk/edit#heading=h.pfs7yv5ul2vw).

If you are interested, there is a slot available on 12th Oct on our [agenda](https://docs.google.com/document/d/1k7VNetgbuDNyIs_87GLQRH2W5SLgjgOhB6pDyv89MYk/edit#heading=h.68c0ynema0xi). Please feel free to add it to the agenda or reach out to us in the [#tag-runtime](https://slack.cncf.io/) channel on the CNCF slack.

Thanks!

cc: @raravena80 @helayoty @nikhita

**Comment by AkihiroSuda** [comment]
> If you are interested, there is a slot available on 12th Oct on our [agenda](https://docs.google.com/document/d/1k7VNetgbuDNyIs_87GLQRH2W5SLgjgOhB6pDyv89MYk/edit#heading=h.68c0ynema0xi).

Thank you, but I'm not sure if I can be available on that day.
When will be the next slot? (26 Oct?)

**Comment by rajaskakodkar** [comment]
@AkihiroSuda yes, next slot is on 26th Oct. We can schedule it then 👍

**Reply by AkihiroSuda** [reply]
Thank you, I booked my slot in the google docs.

**Comment by AkihiroSuda** [comment]
Slides:
- https://github.com/AkihiroSuda/AkihiroSuda/blob/master/slides/2023/20231027%20%5BCNCF%20TAG-Runtime%5D%20Usernetes%20Gen2.pdf
